docs/report/report.tex

```diff
@@ -45,6 +45,92 @@ \subsubsection{Stretch Goals}
 
 \section{Machbarkeitsstudie}
 
+\subsection{NFC Tag}
+\subsubsection{lesen}
+Um mit NFC Tags arbeiten zu können, müssen diese auch auf entschlüsselt bzw. gelesen werden können.
+Hierfür ist ein Hardware NFC-Reader notwendig, der über USB mit dem Chip der Box kommuniziert.
+Hierfür könnte \textit{evdev} genutzt werden, ein Kernel Modul von Linux. Um das mit Node zu nutzen, kann \textit{node-evdev}\footnote{https://github.com/sdumetz/node-evdev} genutzt werden. Mit dem Fork\footnote{https://github.com/anbraten/node-evdev} von Anbraten gibt es auch Raspberry Pi und Typescript support.
+Das ganze emuliert Keyboard Eingaben und sendet die Tag-IDs auf diese Art.
+
+\subsubsection{schreiben}
+Einen NFC-Tag zu beschreiben ist davon abhängig, ob der NFC-Tag über einen eigenen Speicher verfügt.
```

Comment:
unklar.

Reply:
adressed in commit ebddf4dd3e0ad75eae259bd72ee957a80ea9e62b